src/main/java/br/unicamp/cecom/appointmentscheduler/core/features/admin/AdminService.java

```diff
@@ -0,0 +1,66 @@
+package br.unicamp.cecom.appointmentscheduler.core.features.admin;
+
+import br.unicamp.cecom.appointmentscheduler.core.exception.NotFoundException;
+import br.unicamp.cecom.appointmentscheduler.core.features.admin.to.request.CreateAdminRequest;
+import br.unicamp.cecom.appointmentscheduler.core.features.admin.to.request.UpdateAdminRequest;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import javax.persistence.EntityNotFoundException;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
```

Comment:
Add autowired for dependency injection:
```suggestion
@RequiredArgsConstructor(onConstructor = @__(@Autowired))
```